src/connection/connection.ts

```diff
@@ -1,13 +1,20 @@
+<<<<<<< HR-001
+import config from '../config/index';
+
+const mysql = require('mysql');
+
+=======
```

Comment:
You did a merge without resolving conflicts. Consider resolving conflicts first before doing a merge. This code will bug out.